lib/angulartics-mixpanel.js

```diff
@@ -43,6 +43,9 @@ angular.module('angulartics.mixpanel', ['angulartics'])
     $analyticsProvider.registerUserTimings(function (properties, action) {
       mixpanel.time_event(action);
     });
+    $analyticsProvider.registerCookiesClear(function () {
```

Comment:
One problem is this requires a logout handler in the core which doesn't currently exist. These `register*` functions are defined [in an array in the core](https://github.com/angulartics/angulartics/blob/master/src/angulartics.js#L49).